lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -86,15 +92,44 @@ def output_form
       end
 
       def overflow_text
-        return nil unless @has_form4142 || input_form['isTerminallyIll'].present?
-
         overflow = ''
         overflow += TERMILL_OVERFLOW_TEXT if input_form['isTerminallyIll'].present?
         overflow += FORM4142_OVERFLOW_TEXT if @has_form4142
 
+        if Flipper.enabled?(:form526_include_document_upload_list_in_overflow_text)
+          overflow += FORM0781_OVERFLOW_TEXT if @form_content.key?(Form526Submission::FORM_0781)
+          overflow += attached_files_list
+        end
+
         overflow
       end
 
+      def attached_files_list
+        list = ''
+        attachments = input_form['attachments']
+
+        if attachments&.length
+          if attachments.length <= MAX_VETERAN_UPLOADED_FILE_LIST_ATTACHMENTS
+            list += "The veteran uploaded #{attachments.length} documents along with this claim. " \
+                    "Please verify in VBMS eFolder:\n"
+
+            file_guids = attachments&.pluck('confirmationCode')
+            attachment_files = SupportingEvidenceAttachment.where(guid: file_guids)
+            filenames = attachment_files.map(&:original_filename).sort
+
+            filenames.each { |filename| list += "#{filename}\n" }
+          else
+            # If the file list is too long, we only include the message
+            # This is to avoid overfilling the overflowText field
+            # We also don't end the instruction with a colon because we aren't listing filenames after it
```

Comment:
Would ending with a period be better than nothing?

Reply:
Yeah I ended up making that change to simplify this